templates/master/00-master/_base/files/apiserver-url-env.yaml

```diff
@@ -0,0 +1,7 @@
+mode: 0644
+path: "/etc/kubernetes/apiserver-url.env"
+# used by the cluster network operator to learn the URL to the internal apiserver load balancer
```

Comment:
For things like the binaries extracted to the host?  Or code running in a container?  Both?

Reply:
The motivating use case is the CNO, which is in a container (and thus bind-mounts it), but I suspect we'll find a few more uses for it.